crates/storage_impl/src/redis/pub_sub.rs

```diff
@@ -61,120 +77,128 @@ impl PubSubInterface for std::sync::Arc<redis_interface::RedisConnectionPool> {
         logger::debug!("Started on message: {:?}", self.key_prefix);
         let mut rx = self.subscriber.on_message();
         while let Ok(message) = rx.recv().await {
-            logger::debug!("Invalidating {message:?}");
-            let key = match CacheKind::try_from(RedisValue::new(message.value))
-                .change_context(redis_errors::RedisError::OnMessageError)
-            {
-                Ok(value) => value,
-                Err(err) => {
-                    logger::error!(value_conversion_err=?err);
-                    continue;
-                }
-            };
+            let channel_name = message.channel.to_string();
+            match channel_name.as_str() {
+                super::cache::IMC_INVALIDATION_CHANNEL => {
+                    logger::debug!("Invalidating {message:?}");
```

Comment:
Is it sufficient for us to log only the message value, or do we need to log all the fields (channel, server, etc.)?

Reply:
We are logging the entire `Message` struct. this does include all the fields (channel, value, server)